data/en/cffunction.json

```diff
@@ -6,21 +6,21 @@
-		{"name":"output","description":"Specifies under which conditions the function can generate\n HTML output.","required":false,"default":"","type":"boolean","values":[true,false]},
-		{"name":"displayname","description":"Meaningful only for CFC method parameters. A value to be\n displayed in parentheses following the function name when\n using introspection to show information about the CFC","required":false,"default":"","type":"string","values":[]},
-		{"name":"hint","description":"Meaningful only for CFC method parameters. Text to be\n displayed when using introspection to show information\n about the CFC. The hint attribute value follows the syntax\n line in the function description","required":false,"default":"","type":"string","values":[]},
+		{"name":"name","description":"A string; a component method that is used within the cfcomponent tag.","required":true,"default":"","type":"string","values":[]},
+		{"name":"returntype","description":"String; a type name; data type of the function return value. Required for a web service; optional otherwise.\n","required":false,"default":"any","type":"string","values":["any","array","binary","boolean","date","guid","numeric","query","string","struct","UUID","(variable name)","void","xml","(component name)"]},
+		{"name":"roles","description":"A comma-delimited list of CFML security roles that can invoke the method. Only users who are logged in with the specified roles can execute the function. If this attribute is omitted, all users can invoke the method.","required":false,"default":"","type":"string","values":[]},
+		{"name":"access","description":"The client security context from which the method can be invoked.\n\n private: Available only to the component that declares the method and any components that extend the component in which it is defined.\n\n package: Available only to the component that declares the method, components that extend the component, or any other components in the package.\n\n public: Available to a locally executing page or component method.\n\n remote: Available to a locally or remotely executing page or component method, or a remote client through a URL, Flash, or a web service. To publish the function as a web service, this option is required.\n","required":false,"default":"public","type":"string","values":["private","package","public","remote"]},
+		{"name":"output","description":"Specifies under which conditions the function can generate HTML output.\n","required":false,"default":"","type":"boolean","values":[true,false]},
+		{"name":"displayname","description":"Meaningful only for CFC method parameters. A value to be displayed in parenthesis following the function name when using introspection to show information about the CFC.","required":false,"default":"","type":"string","values":[]},
```

Comment:
Shouldn't this stay as _parentheses_? 

I am certainly not an expert in this area but it appears that parentheses is plural and parenthesis is singular. https://english.stackexchange.com/questions/130219/parentheses-vs-parenthesis